docs/tutorial/acquiring_data/hek.rst

```diff
@@ -281,23 +263,18 @@ Let's say we want all the flares west of 50 arcseconds AND have a peak flux over
    >>> result = Fido.search(a.Time(tstart,tend), a.hek.EventType(event_type), (a.hek.Event.Coord1 > 50) and (a.hek.FL.PeakFlux > 1000))  # doctest: +REMOTE_DATA
 
    >>> result["hek"]["fl_peakflux"] # doctest: +REMOTE_DATA
-   <QueryResponseColumn name='fl_peakflux' dtype='float64' length=7>
-   2326.86
-   1698.83
-   2360.49
-   3242.64
-   1375.93
-   6275.98
-   1019.83
-   >>> result["hek"]["event_coord1"] # doctest: +REMOTE_DATA
-   <QueryResponseColumn name='event_coord1' dtype='float64' length=7>
-   883.2
-   883.2
-   883.2
-   883.2
-   883.2
-   883.2
-   883.2
+   <MaskedQuantity [2326.86, 1698.83, 2360.49, 3242.64, 1375.93, 6275.98,
+                     1019.83] DN / (pix s)>
+
+   >>> result["hek"]["event_coord"] # doctest: +REMOTE_DATA
+   <QueryResponseColumn name='event_coord' dtype='object' length=7>
```

Comment:
Why is this output not a single SkyCoord column? they are all in the same frame and have the same shape no?

Reply:
No, they are not in the same frame. Each row can potentially have a different frame as determined by the "event_coordsys" column. I suppose we could convert the whole column to the frame of the first entry or something like that.

Reply:
Hmm, I guess it's better to be consistent rather than return a single SkyCoord if they happen to be the same frame like in this example?

Reply:
I don't think we should transform fwiw.